app/helpers/createAuthProvider.js

```diff
@@ -0,0 +1,15 @@
+const { AuthProvider, ErrorContext } = require('buying-catalogue-library');
+const { logger } = require('../logger');
+
+export const createAuthProvider = ({ config }) => {
+  const unauthenticatedError = new ErrorContext({
+    status: 401,
+    title: 'You\'re not authorised to view this page',
+    description: 'You must be logged in as an admin USER to access Buying Catalogue admin.',
```

Comment:
not with capitals 🤣 I wrote it like that to make it obvious what I changed

Reply:
😂